ionoscloud/resource_dbaas_pgsql_cluster.go

```diff
@@ -68,6 +68,7 @@ func resourceDbaasPgSqlCluster() *schema.Resource {
 				MaxItems:    1,
 				Description: "Configuration options for the connection pooler",
 				Optional:    true,
+				Computed:    true,
```

Comment:
if this changed, we should add it to the changelog

Reply:
@cristiGuranIonos yes, I forgot to add it in the `CHANGELOG.md`. This was supposed to be done in the PR for the `connection_pooler` attribute but I missed it. I'll add it in both `CHANGELOG.md` and title.

Reply:
@cristiGuranIonos done in e8383e7f036a9ff9cae95effc13bfa7c0eb37daa